apis/core.oam.dev/common/types.go

```diff
@@ -196,7 +196,7 @@ type RawComponent struct {
 type AppStatus struct {
 	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
-	v1alpha1.RolloutStatus `json:",inline"`
+	AppRolloutStatus `json:",inline"`
```

Comment:
```suggestion
	Rollout *AppRolloutStatus `json:"rollout,omitempty"`
```

This is an incompatible change but we must do so.